src/components/Sidebar/Sidebar.js

```diff
@@ -21,6 +21,7 @@ const Sidebar = styled.div`
 `;
 
 const SidebarMain = styled.div`
+  flex: 1 1 0;
```

Comment:
Could you please have some comment why do you need to set `flex-basis` to 0 and auto to make scrollbar in sidebar work?

This can be useful for other people who will work on this area.

Reply:
This article can explain that better https://mastery.games/post/the-difference-between-width-and-flex-basis/?fbclid=IwAR0KwkNZsn_Wp_8NZ8vlo9Hxn1TKZwPHRQ9aygCG8wVZmQl7e6ViLc9xqCM.

Reply:
Very minor, could you please remove FB query string from the link.